pkg/templates/templates/infinispan-base-16.xml

```diff
@@ -0,0 +1,18 @@
+{{template "header.xml" . }}
+
+<cache-container name="default" statistics="true">
```

Comment:
The content of this file is the same as `infinispan-base-15.xml`. We should be able to split the content of `infinispan-base-15.xml` into common templates and then call them here.

We could create the following two templates:

- `templates/common/cache-container.xml`
- `templates/common/server.xml`

It should the be possible to make `infinispan-base-15.xml` and `infinispan-base-16.xml` simply be:

```
{{template "header.xml" . }}

{{template "cache-container.xml" . }}
{{template "server.xml" . }}
```